src/client.ts

```diff
@@ -179,11 +181,13 @@ export class Client {
   /**
    * Initialises OpenTelemetry instrumentation
    */
-  private initOpenTelemetry() {
+  private initOpenTelemetry(
+    additionalInstrumentations?: OpenTelemetryOptions["additionalInstrumentations"]
+  ) {
     const sendParams = this.config.data.sendParams
     const sendSessionData = this.config.data.sendSessionData
     const sdk = new NodeSDK({
-      instrumentations: [
+      instrumentations: (additionalInstrumentations ?? []).concat([
```

Comment:
I would turn this around, let `additionalInstrumentation` overwrite our defaults. That way, if users want to, they override the HttpInstrumentation or PrismaInstrumentation (and any others) with their own config. How it's currently implemented any conflicting additionalInstrumentation is silently ignored.